cypress/integration/sample_spec.ts

```diff
@@ -0,0 +1,5 @@
+describe('The Home Page', () => {
+  it('sucessfully loads', () => {
+    cy.visit('/');
```

Comment:
Could you had one basic check like there is some DOM element we know should be there or something more cool like checking Cesium globe loaded.

Reply:
Added declining the tracking consent.

Unfortunately i think what takes long is not DOM but content, also the page has infinite reloads in the cypress test environment and requires a hard coded wait to execute the test more stable.

Reply:
Hmm, I see. Definitely not something to enable in CI then.
Maybe Cypress is not completly ready for us yet.